easybuild/easyconfigs/m/MEGA/MEGACC-7.0.18-1.eb

```diff
@@ -0,0 +1,33 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2016, UNIGE
+# Authors::   Yann Sagon <yann.sagon@unige.ch>
+# License::   MIT/GPL
+# $Id$
+##
+
+name = 'MEGACC'
+
+version = '7.0.18-1'
+
+easyblock = 'Tarball'
+
+homepage = 'http://www.megasoftware.net'
+description = """MEGA-Computing Core - Sophisticated and user-friendly software suite for analyzing DNA and protein sequence data from species and populations."""
```

Comment:
@ysagon lines should be shorter than 120 characters, please wrap this line like so:

```python
description = """MEGA-Computing Core - Sophisticated and user-friendly software suite for analyzing DNA and
 protein sequence data from species and populations."""
```